pages/decision-reviews/board-appeal/hearing-with-veterans-law-judge.md

```diff
@@ -0,0 +1,195 @@
+<a href="/decision-reviews/board-appeal/">Learn more about requesting a Board Appeal for decisions received on or after February 19, 2019</a>.
+
+</div>
+</li>
+
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="before">If you received your decision before February 19, 2019</button>
+<div id="before" class="usa-accordion-content">
+
+When you fill out the Appeal to Board of Veterans' Appeals (VA Form 9), select one of the options for a hearing in block 10.<br>
+
+<b>The form will ask you to choose 1 of these 4 options:</b>
+<ul>
+  <li><b>No hearing.</b> If you choose this option, you can send a letter to the Board that delivers the same message as what you would have said at a hearing.</li>
+  <li><b>A videoconference hearing at your local VA office.</b> This will take place at your local VA office (or another facility close to you). The Board can usually schedule video hearings sooner than in-person hearings.<br>
+<a href="https://www.bva.va.gov/docs/BVA-VideoHearing-508version.pdf">Find out more about choosing a video hearing</a>.</li>
+  <li><b>An in-person hearing at your local VA office.</b> This can sometimes delay your hearing, because the Board has to schedule judges to travel to your local VA office.</li>
+  <li><b>An in-person hearing at the Board in Washington, DC.</b></li>
+
+<a href="https://www.va.gov/vaforms/va/pdf/va9.pdf">Download VA Form 9</a>.<br>
+<a href= "/disability/file-an-appeal/">Learn more about requesting a Board Appeal for decisions received before February 19, 2019</a>.
+</div>
+</li>
+</ul>
+
+### How will I know when my hearing is scheduled?
+
+You’ll receive a notice in the mail at least 30 days before your scheduled hearing. The notice will list the date, time, and location of your hearing.
+
+<span id="prepare-for-board-hearing"></span>
+## Prepare for your Board hearing
+
+### Can I bring evidence to my hearing?
+
+Yes. You can bring new evidence to your hearing to share with the judge. But please don't submit new evidence before your hearing.
+
+**If you received your decision on or after February 19, 2019,** according to the new process, you can also submit new evidence for the judge's review for up to 90 days after your hearing. 
+
+### Who can help me prepare for my hearing?
+
+An accredited representative who’s trained in the appeals process can help you prepare. Your representative must be a VA-accredited lawyer, claims agent, or someone from a Veterans Service Organization (VSO).<br>
+
+[Get help preparing for your hearing](/decision-reviews/get-help-with-review-request/).
+
+<span id="what-to-expect-at-board-hearing"></span>
+## Find out what to expect at your Board hearing
+
+### What will happen at my hearing?
+
+To start, the judge will ask you to take an oath that you’ll tell the truth during the hearing.
+
+Then you and the judge will have a conversation. The judge may ask you a few questions, but don't worry—it won’t be like a cross-examination. You'll:
```

Comment:
Hey @bethpotts, this list is rendering funny because of a `usa-accordion` on [line 38]9https://github.com/department-of-veterans-affairs/vagov-content/pull/304/files#diff-e1f027cfc54840c6c0bb6042b0f9867bR38). I think that class should be removed?